ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.ts

```diff
@@ -363,16 +369,34 @@ export class ArchivesLoggingFiltersComponent implements OnInit, OnChanges, OnDes
     setDefaultPublishDateFilter() {
         const defaultPublishDateInterval = ConfigService.getConfigValue('archive.filters.publishDate.days', 10);
 
-        const min = new Date();
-        const minDate = sub(min, {days: defaultPublishDateInterval});
+        const currentDate = new Date();
+        this.defaultPublishMaxDate = new Date(
+            currentDate.getFullYear(),
+            currentDate.getMonth(),
+            currentDate.getDate(),
+            23,
+            59,
+            59
+        );
+
+        this.defaultPublishMinDate = new Date(
+            currentDate.getFullYear(),
+            currentDate.getMonth(),
+            currentDate.getDate(),
+            0,
+            0,
+            0
+        );
+        this.defaultPublishMinDate = sub(this.defaultPublishMinDate, {days: defaultPublishDateInterval});
 
-        this.defaultMinPublishDateStringFormat =
-            minDate.getFullYear() +
-            '-' +
-            String(minDate.getMonth() + 1).padStart(2, '0') +
-            '-' +
-            String(minDate.getDate()).padStart(2, '0') +
-            'T00:00';
+        this.publishDateRangeSelected = {startDate: this.defaultPublishMinDate, endDate: this.defaultPublishMaxDate};
+    }
+
+    onDateRangeChange(): void {
+        this.publishMinDate = this.parentForm.value.publishDateRange.startDate;
+        this.publishMaxDate = this.parentForm.value.publishDateRange.endDate;
+        this.activeMinDate = this.parentForm.value.activeDateRange.startDate ?? null;
+        this.activeMaxDate = this.parentForm.value.activeDateRange.endDate ?? null;
```

Comment:
I think you do not need anymore these four variables as the range is fully managed by the date range picker component